.travis.yml

```diff
@@ -36,9 +44,7 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
```

Comment:
I think bower should still be installed here

Reply:
Thanks! It's getting greener https://travis-ci.org/workmanw/ember-string-ishtmlsafe-polyfill/builds/358296726

Reply:
I'm following the build as well ;-)

Reply:
Exciting Mondays ;)